app/views/container_group/show.html.haml

```diff
@@ -1,6 +1,4 @@
-- if %w(containers container_projects).include?(@display)
-    = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
-- elsif @display == 'container_services' && @showtype != "compare"
+- if %w(container_services containers).include?(@display)
```

Comment:
@h-kataria Is it intentional that @showtype != "compare" was only for container_services
and not for containers? should I maintain it?